### ISSUE TITLE
MONDO:0022652 cardiomyopathy dilated with conduction defect type 2 [Obsolete]

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#2369]
**Mondo term (ID and Label)**
MONDO:0022652; cardiomyopathy dilated with conduction defect type 2

**Reason for deprecation**
GARD is the only cross-reference for this concept and it appears to be a bit of an "orphan" term (not in a logical hierarchy) 
and the GARD page seems only to refer back to the Monarch Initiative ... https://rarediseases.info.nih.gov/diseases/5644/cardiomyopathy-dilated-with-conduction-defect-type-2

**Suggested term to consider**
dilated cardiomyopathy 1E MONDO:0011003 Seems to encompass a similar concept "dilated cardiomyopathy with conduction defect 2"

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Comment by nicolevasilevsky** [comment]
cc @galyea-gard

**Comment by nicolevasilevsky** [comment]
pinging @galyea-gard, please let me know what you think about obsoleting this term, thanks!

**Comment by galyea-gard** [comment]
I do agree with the merging. Also, you are right that “cardiomyopathy dilated with conduction defect type 2” is a synonym of  “dilated cardiomyopathy 1E”.  I made the changes in GARD (so now we have “dilated cardiomyopathy 1A” and “dilated cardiomyopathy 1E” as the main terms for these disorders). However, I believe we won't have these subtypes when our review process is finished. 
Note:  These terms were added in GARD back in 2013 and there was no ID in the comments, so I can’t tell what the source was. Thank you!
